### ISSUE TITLE
DT unexpected freeze all system

### ISSUE THREAD

**Issue by tomwiet** [darktable-org/darktable#6032]
Hi
DT v 3.2.1 freeze system. This happens at various times during editing without a clear pattern. Sometimes when loading a photo in darkroom, sometimes after use various modules. There is no pattern here.
My system: Debian Bullseye, kernel SMP Debian 5.7.10-1 (2020-07-26) x86_64 GNU/Linux, 
Hardware: Inetel NUC 7i7BNH (i7-7567U with Iris Plus 650)
Attache some debig log after run TD with -d all parameter.
[dt-2020-08020.log](https://github.com/darktable-org/darktable/files/5101465/dt-2020-08020.log)
[dt-2020-08-20-2.log](https://github.com/darktable-org/darktable/files/5101466/dt-2020-08-20-2.log)
[dt-2020-08-20-1.log](https://github.com/darktable-org/darktable/files/5101467/dt-2020-08-20-1.log)

**Comment by parafin** [comment]
If it freezes the whole system then it's not a darktable bug. Try disabling OpenCL.

**Comment by tomwiet** [comment]
> If it freezes the whole system then it's not a darktable bug. Try disabling OpenCL.

I tried run with and without OpenCL. Same effect.

**Comment by johans3** [comment]
try with another branch, debian is ok but i 've never heard about "bulls…"

**Comment by tomwiet** [comment]
> try with another branch, debian is ok but i 've never heard about "bulls…"

Bullseye is codename of actual testing distribution of Debian. 
Actualy i removed i915 firmware, reset DT configuration and try this. Its posibble as wrote parafin, that is not DT issue. Unfortunately Intel cant remove some bug on his linux i915 driver and maybe it is a problem?

**Comment by maboleth** [comment]
Sadly I cannot help you here entirely, but i've seen several reports regarding intel chip. Do you have the latest microcode installed during boot?

Sys crash/freeze shouldnt be possible by DT.

**Comment by blitzgneisserin** [comment]
same problem on Debian testing/sid. I think this does not happen on other Linux systems. However, I have no idea how to produce a backtrace.
Hopefully this will be fixed in Debian. However... I am not sure whether is is a Debian or a darktable bug. I did have this issue in early spring, then there was a pause for some months, and these crashes came back in July I think. There was definitely a significant break, i.e. a period during which I did not experience any crashes.
